arlas-tests/src/test/resources/results/_describe_base_result.json

```diff
@@ -91,6 +91,93 @@
           "type": "GEO_SHAPE",
           "taggable": false,
           "indexed": true
+        },
+        "h3": {
+          "type": "OBJECT",
+          "properties": {
+            "0": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "1": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "2": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "3": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "4": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "5": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "6": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "7": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "8": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "9": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "10": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "11": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "12": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "13": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "14": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            },
+            "15": {
+              "type": "KEYWORD",
+              "taggable": false,
+              "indexed": true
+            }
+          },
+          "taggable": false,
+          "indexed": true
```

Comment:
Is this correct to have H3 indexed and taggable? isn't it an intermediate json node?

Reply:
I tried this mapping:
```json
        "h3": {
          "enabled": false,
          "properties": {
            "0": {
              "type": "keyword",
              "index": true
            },
```
expecting
```
...h3={indexed=false, taggable=false, type=OBJECT, properties={11={indexed=true, taggable=false, type=KEYWORD}...
```
as a result but got 
```
...h3={indexed=false, taggable=false, type=OBJECT, properties={11={indexed=false, taggable=false, type=KEYWORD}...
```

so unless there is another way to skip the parent node, I fear it is not possible.